README.md

```diff
@@ -44,7 +44,7 @@ public function fields(Request $request){
 | `dependsOn`                   | String, String  | null, null | This method allows you to depend on belongsto field, this make an auto query                                                                                                 |
 | `canSelectAll`                | String, Boolean | 'Select All', true | This method allows you to have a select all checkbox and display custom message                                                                                              |
 | `showAsListInDetail`          | Boolean         | true       | This method allows you to display as list in detail                                                                                             |
-
+| `language`                    | String          | Current Locale       | This method allows for the correct display of translatable models |
```

Comment:
Uses \` App::getLocale()\` to obtain the current language to be used as the default.